src/assets/locales/resources_en_US.json

```diff
@@ -1,6 +1,66 @@
 {
-  "skyux_some_key": {
+  "file_item_delete": {
```

Comment:
Sorry I missed this before, but I've been prefixing the resource keys with `skyux_` since we're making these "public" and not internal. Also wouldn't hurt to put the component name in there, too.

For example:
```
"skyux_file_attachment_file_size_b_plural": {
  "_description": "",
  "message": "{0} bytes"
}
```

See: https://github.com/blackbaud/skyux-core/blob/master/src/assets/locales/resources_en_US.json#L2

I've created an issue to change the prefix for the indicators, as well: https://github.com/blackbaud/skyux-indicators/issues/9

Reply:
this is done.